presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java

```diff
@@ -92,4 +94,17 @@ public NodeState getServerState()
             return ACTIVE;
         }
     }
+
+    @GET
+    @Path("coordinator")
+    @Produces(TEXT_PLAIN)
+    public Response getServerCoordinator()
```

Comment:
I am really needing a 200 series http code returned from the coordinator.  I have not had any luck so far with our LBs with string parsing or 300/400 series http return codes.